projects/lights_control/test/test_lights_gpio.c

```diff
@@ -129,14 +129,14 @@ void test_lights_gpio_set_rear() {
 
 void test_get_lights_board_front() {
   TEST_ASSERT_OK(lights_gpio_init(LIGHTS_GPIO_INIT_MODE_TEST_FRONT));
-  LightsBoard board;
+  volatile LightsBoard board;
```

Comment:
The build was failing when I didn't have this `volatile` here. Here was the message I was getting: 
```
projects/lights_control/test/test_lights_gpio.c:132:15: error: variable 'board' might be clobbered by 'longjmp' or 'vfork' [-Werror=clobbered] 
   LightsBoard board;                                                                                                                          
               ^ 
```
What is this? 😕 
What I found from `gcc` documentation: 

> Warn for variables that might be changed by longjmp or vfork. This warning is also enabled by -Wextra.